Sources/Apodini/Semantic Model Builder/PathModelBuilder.swift

```diff
@@ -71,7 +71,7 @@ private struct PathComponentElementParser: PathComponentParser {
 
 // MARK: Helpers
 
-private extension Parameter {
```

Comment:
I think this extension can remain private since it is only getting used in the scope of this file